charts/enterprise-metrics/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 1.7.2
+version: 1.7.3
 appVersion: v1.6.1
 description: "Grafana Enterprise Metrics"
 engine: gotpl
```

Comment:
Will want to make sure we update this appropriately once the new version of GEM is released.